lib/fable/events.ex

```diff
@@ -242,13 +242,20 @@ defmodule Fable.Events do
         ) ::
           transation_fun
   def emit(config, %{__meta__: %Ecto.Schema.Metadata{}} = aggregate, fun, changes \\ %{}, opts)
-      when is_function(fun, 3) do
+      when is_function(fun, 3) or is_function(fun, 2) do
```

Comment:
What do you think about an explicit error in the case statement and not even having the guards? No need to do the checks twice.

Reply:
I think tossing the guards is fine, it doesn't appear that they are actually being used as branching logic.